File: PHP/car.php
<?php

class Car {
  public $id;
  public $license;
  public $driver;
  protected $passenger;

  public function __construct($license, $driver) {
    $this->license = $license;
    $this->driver = $driver;
  }

  public function PrintDataCar(){
    echo "license: $this->license, conductor: {$this->driver->name}, document: {$this->driver->document}";
  }


  public function getPassanger() {
    return $this -> passanger;
  }

  public function setPassanger($passanger) {
    if ($passanger == 4) {
      $this -> passanger = $passanger;
    }

    else {
      echo "Necesita asignar 4 pasajeros";
    }
  }
}

File: Java/Car.java
package Java;

public class Car {
    private Integer id;
    private String license;
    private Account driver;
    protected Integer passenger;

    public Car (String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    public Integer getPassanger(){
        return passenger;
    }

    public void setPassanger(Integer passanger){

        if( passanger == 4) {

            this.passenger = passanger;

        }
        else {
            System.out.println("Necesita asignar 4 pasajeros");
        }
        
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }
    
    
}
